Dockerfile

```
@@ -1,36 +1,42 @@
-FROM ghcr.io/astral-sh/uv:python3.13-bookworm
+# Lightweight multi-stage build for container optimization
+# Stage 1: Build dependencies using uv
+FROM ghcr.io/astral-sh/uv:python3.13-bookworm AS builder
 
 WORKDIR /app
 
-# Enable bytecode compilation
+# Enable bytecode compilation for better performance
 ENV UV_COMPILE_BYTECODE=1
-
 # Copy from the cache instead of linking since it's a mounted volume
 ENV UV_LINK_MODE=copy
 
+# Install dependencies first (better layer caching)
 RUN --mount=type=cache,target=/root/.cache/uv \
     --mount=type=bind,source=uv.lock,target=uv.lock \
     --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
     uv sync --locked --no-install-project --no-dev
 
-# Installing separately from its dependencies allows optimal layer caching
-COPY  /app /app
+# Copy and install the application
+COPY app /app
 RUN --mount=type=cache,target=/root/.cache/uv \
     --mount=type=bind,source=uv.lock,target=uv.lock \
     --mount=type=bind,source=pyproject.toml,target=pyproject.toml \
     uv sync --locked --no-dev
 
-ENV PATH="/app/.venv/bin:$PATH"
+# Stage 2: Lightweight runtime using Alpine Linux (45MB vs 1GB base)
+FROM python:3.13-alpine AS runtime
 
-WORKDIR /
+WORKDIR /app
```

```suggestion
WORKDIR /
```

ルートからの起動を前提にしているのでこのように直して下さい